src/components/Karma/CircleGradient.tsx

```diff
@@ -11,7 +11,7 @@ export const CircleGradient = (props: {
   children: React.ReactNode;
   testId?: string;
 }) => {
-  const r = props.size / 2 - props.width / 1.9;
+  const r = props.size / 2 - props.width;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The radius calculation has been changed from `props.size / 2 - props.width / 1.9` to `props.size / 2 - props.width`. This change will affect the size of the circle. If this is intended, no action is needed. However, if the original size was correct, you should revert this change.
```diff
- const r = props.size / 2 - props.width;
+ const r = props.size / 2 - props.width / 1.9;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->